pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -205,6 +205,12 @@ func (rc *reconciler) unmountVolumes() {
 			err := rc.operationExecutor.UnmountVolume(
 				mountedVolume.MountedVolume, rc.actualStateOfWorld, rc.kubeletPodsDir)
 			if err != nil && !isExpectedError(err) {
+				pod := &v1.Pod{
```

Comment:
Why is the FailedMountVolume reporting in the operation, but this is in the reconciler? I feel like it will be easier to update and maintain if the error reporting is all in the same place.

Reply:
@mattcary thx, I find that there are several places to report FailedMountVolume, 2 in operation, 1 in kubelet main, I write the code just like kubelet main:
pkg/kubelet/kubelet.go
```
func (kl *Kubelet) syncPod(ctx context.Context, updateType kubetypes.SyncPodType, pod, mirrorPod *v1.Pod, podStatus *kubecontainer.PodStatus) (isTerminal bool, err error) {
	if !kl.podWorkers.IsPodTerminationRequested(pod.UID) {
		// Wait for volumes to attach/mount
		if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
			kl.recorder.Eventf(pod, v1.EventTypeWarning, events.FailedMountVolume, "Unable to attach or mount volumes: %v", err)
			klog.ErrorS(err, "Unable to attach or mount volumes for pod; skipping pod", "pod", klog.KObj(pod))
			return false, err
		}
	}
```

Reply:
@mattcary ok, move to operation :)